### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by RezaOptic**
## Description
add ```MINIO_ACCESS_KEY``` and ```MINIO_SECRET_KEY``` environment variables to docker run commands

### CODE REVIEW

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :x: [more...](#60e2c8d1-fd1e-4000-bef0-5f41436f35db) |
<a name="60e2c8d1-fd1e-4000-bef0-5f41436f35db"></a>
### 9369-10a871a/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev3.minio.io:30049
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 846fe0e8697e:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 8 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 1 minutes and 56 seconds
(4/15) Running aws-sdk-ruby tests ... done in 15 seconds
(5/15) Running awscli tests ... FAILED in 3 seconds
{
  "name": "awscli",
  "duration": 1427,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "An error occurred (BucketAlreadyOwnedByYou) when calling the CreateBucket operation: Your previous request to create the named bucket succeeded and you already own it."
}

Executed 4 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Review by kannappanr — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ MinIO is High Performance Object Storage released under Apache License v2.0. It
 ### Stable
 ```
 docker pull minio/minio
-docker run -p 9000:9000 minio/minio server /data
+docker run -e MINIO_ACCESS_KEY=samplekey -e MINIO_SECRET_KEY=samplekey -p 9000:9000 minio/minio server /data
```
Comment (harshavardhana) [inline README.md:12]: There is a note right below, is there a problem with that?

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RezaOptic) [reply README.md:12]: by default these keys are empty, and after we are running docker image in the first page of minio we must enter these keys, and if we do not set this keys, we cant enter keys and can't use the API's

Comment (harshavardhana) [reply README.md:12]: It is indicated right after this line as a NOTE: on what needs to be done, I don't think it's appropriate to add this documentation here @RezaOptic

**Pull request closed without merging**